src/components/applications/ApplicationFilter.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { Filter, TextInput } from "react-admin";
+
+const ApplicationFilter = (props) => (
```

Comment:
This can be turned into a reusable component. We'd just need to pass in an object that represents the filter options.

```
{
    label: string,
    source: string,
    alwaysOn?: boolean, (default = false or true)
}
```

Reply:
all done now